include/mbedtls/threading.h

```diff
@@ -97,6 +97,9 @@ extern int (*mbedtls_mutex_unlock)( mbedtls_threading_mutex_t *mutex );
  */
 extern mbedtls_threading_mutex_t mbedtls_threading_readdir_mutex;
 extern mbedtls_threading_mutex_t mbedtls_threading_gmtime_mutex;
+#if defined(MBEDTLS_ECP_INTERNAL_ALT)
+extern mbedtls_threading_mutex_t mbedtls_threading_ecp_mutex;
```

Comment:
Mutexing should be done by the underlying hardware implementation, in line with the other alternate core implementations. If the hardware is capable of multiple concurrent ECP operations, mbedTLS shouldn't prevent that.